File: apps/frontend/src/modules/clientes/types/cliente.ts
export interface Cliente {
  id: number;
  nome?: string | null;
  fantasia?: string | null;
  documento?: string | null;
  inscricaoEstadual?: string | null;
  telefone?: string | null;
  celular?: string | null;
  email?: string | null;
  cep?: string | null;
  endereco?: string | null;
  numero?: string | null;
  complemento?: string | null;
  bairro?: string | null;
  cidade?: string | null;
  uf?: string | null;
  situacao?: string | null;
  observacoes?: string | null;
}

export type ClienteInput = Omit<Cliente, 'id'> & { id?: number };

File: apps/backend/Entities/Cliente.cs
using Gestor.Api.Common;

namespace Gestor.Api.Entities;

public class Cliente : IEntity
{
    public int Id { get; set; }
    public string? Nome { get; set; }
    public string? Fantasia { get; set; }
    public string? Documento { get; set; }
    public string? InscricaoEstadual { get; set; }
    public string? Telefone { get; set; }
    public string? Celular { get; set; }
    public string? Email { get; set; }
    public string? Cep { get; set; }
    public string? Endereco { get; set; }
    public string? Numero { get; set; }
    public string? Complemento { get; set; }
    public string? Bairro { get; set; }
    public string? Cidade { get; set; }
    public string? Uf { get; set; }
    public string? Situacao { get; set; }
    public string? Observacoes { get; set; }
}
